packages/cozy-stack-client/src/DocumentCollection.js

```diff
@@ -493,7 +517,7 @@ The returned documents are paginated by the stack.
    * @returns {MangoQueryOptions} Mango options
    */
   toMangoOptions(selector, options = {}) {
-    let { sort, indexedFields, partialFilter } = options
+    let { sort, indexedFields, partialFilter, partialIndex } = options
```

Comment:
It seems there is a confusion between partialFilter and partialIndex here, this is supposed to be the same thing (more precisely, a partialFilter is the actual definition of a partialIndex).